controllers/imagejob/imagejob_controller.go

```diff
@@ -108,10 +125,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		return ctrl.Result{}, err
 	}
 
-	// map of node names and runtime
-	nodeMap := processNodes(nodes.Items)
-
-	for nodeName, runtime := range nodeMap {
+	for _, n := range nodes.Items {
+		n := n
+		nodeName := n.Name
+		runtime := n.Status.NodeInfo.ContainerRuntimeVersion
 		runtimeName := strings.Split(runtime, ":")[0]
 		mountPath := getMountPath(runtimeName)
 		if mountPath == "" {
```

Comment:
can you move the log line below this to controllerLog

Reply:
which log line?